e2e_samples/parking_sensors_synapse/src/ddo_transform/.devcontainer/Dockerfile

```diff
@@ -39,4 +52,7 @@ ENV DEBIAN_FRONTEND=dialog
 ENV PACKAGE_VERSION=localdev
 
 # Set the default shell to bash rather than sh
-ENV SHELL /bin/bash
\ No newline at end of file
+ENV SHELL /bin/bash
+
+# Set user to non-root user to get rid of permission errors
+USER $USERNAME
```

Comment:
1. We need to update docker-compose , devcontainer.json as well. And docker-compose should specify the user so that mounted workspace usages the same permission.
2. I think it will be good if switch to $USERNAME before we copy the files otherwise still there will some files owner by root.
3. We need to create virtual env in a location which is owned by $USERNAME otherwise in devcontainer we will not be able to add new dependencies without switching to root first.